examples/Console/Program.cs

```diff
@@ -54,7 +54,7 @@ public static void Main(string[] args)
 internal class ZipkinOptions
 {
     [Option('u', "uri", HelpText = "Please specify the uri of Zipkin backend", Required = true)]
-    public string Uri { get; set; }
+    public string? Uri { get; set; }
```

Comment:
This caught my attention because line 56 says `Required = true`.
However, I think this is the best fix for this line.

This class in instantiated by the ComandLineParser on line 34. 

The correct fix would be to include this in a constructor.